### ISSUE TITLE
Archived Chats Popup is Inaccessible to VoiceOver and Can Trap a User

### ISSUE THREAD

**Issue by Simon818** [TelegramMessenger/Telegram-iOS#1154]
### Checklist
- ☑ I am reporting an issue in existing functionality that does not work as intended
- ☑ I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
Describe the issue that you are experiencing

When accessing archived chats for the first time after a recent update, a screen is shown with information about how archived chats work. This screen is not accessible at all to Voiceover.

### Expected Behavior
Tell us what should happen

VoiceOver should read the text on this screen as well as the button labels. It should not be possible to navigate outside this screen and read chats that are not visually present on the screen.

### Actual Behavior
Tell us what happens instead

VoiceOver does not read the text of this screen or the button labels. The user will have no idea what this screen is trying to tell them or how to dismiss it. VoiceOver users can also swipe left a few times and reach the list of archived chats even though they are visually obscured.

### Steps to Reproduce
1. Open archived chats on a newly installed version of Telegram for iOS
2. Make sure the intro screen is visible. Mine talks about how archived chats work and has a "GOT IT" button.
3. Load VoiceOver and try to read anything at all on this screen. Swipe left and right to navigate around.

### Screenshots and Videos

I have already fumbled my way off this screen and can't get it back again; apologies.

### Environment

**Device:** iPhone 13

**iOS version**: 16.6

**App version:** 9.6.5 (26567)
